src/Node/Crypto/Hash.purs

```diff
@@ -48,6 +62,14 @@ hash alg str enc = do
   buf <- fromString str UTF8
   createHash alg >>= flip update buf >>= digest >>= toString enc
 
+hashBuffer
+  :: Algorithm
+  -> Buffer
+  -> Encoding
+  -> Effect String
+hashBuffer alg buf enc = do
+  createHash alg >>= flip update buf >>= digest >>= toString enc
```

Comment:
It looks like this function isn't needed.
You can change `hash` function to receive `Buffer` instead of `String`.

Reply:
Are you sure you want to change your API by changing the arg type for hash?
Existing users will have to change their code. 
Why not just create a hash' function to match the other functions?

Reply:
@jsparkes 
It's ok to change the interface of `hash`.
You have to check  exports.
`hash` is a private function.
Existing user can not touch `hash`.
There is no problem as long as the interface of exported functions are not changed.